R/utils.R

```diff
@@ -200,3 +200,47 @@ ensure_data.table <- function(data) {
   }
   return(data)
 }
+
+#' @title Print information about a `forecast` object.
+#' @description This function prints information about a forecast object,
+#' including "Forecast type", "Score columns",
+#' "Forecast unit".
+#' @export
+#' @keywords print-forecast
```

Comment:
Ah the pkgdown test is complaining about this line because the topic must be included in the reference list in _pkgdown.yaml. I suggest changing it to one of the keywords we already have and revisiting later when we overhaul the pkgdown index. 

```suggestion
#' @keywords check-forecast
```